### ISSUE TITLE
deployed application does not find definition

### ISSUE THREAD

**Issue by tienbuide** [nickcharlton/urbanscraper#60]
the on heroku deployed application does not find any definitions, eventhough there exists some.

example:
```sh
$ curl https://urbanscraper.herokuapp.com/define/wtf
{"message":"No definitions could be found for: wtf"
```

ref: https://www.urbandictionary.com/define.php?term=wtf

**Comment by nickcharlton** [comment]
That's almost certainly a change with UrbanDictionary and the HTML it's trying to pull the value out of it. I'll take a look this weekend and see if I can work out what's changed.

**Comment by tienbuide** [comment]
@nickcharlton can you please take a look on the PR? thanks :)

**Comment by nickcharlton** [comment]
I just merged #67 and deployed, so this is now fixed.
